docs/troubleshooting/troubleshooting.md

```diff
@@ -0,0 +1,112 @@
+# Troubleshooting
+
+Sometimes you hit a roadblock. That's natural, in coding.
+
+Below are a few problems our users have seen, and ways they solved them in the
+past. If what you're looking for is not on this page, let us know at
+[help@streamlit.io](mailto:help@streamlit.io).
+
+## First things to try...
+
+We try to fix bugs quickly, so many times a problem will go away when you
+upgrade Streamlit. So the first thing to try when having an issue is upgrading
+to the latest version of Streamlit:
+
+```bash
+$ pip install --upgrade streamlit
+$ streamlit version
+```
+
+...and then verify that the version number printed is `0.50.2`.
```

Comment:
Do we update this every time we push a new version?

Reply:
Good question. I haven't been updating this on a regular cadence. Let me see what's there today. Maybe this is something someone on the team scrubs for with each release?

Reply:
So looking through the doc it currently has the version number listed as `0.51.0`. This looks accurate per what's on the site, but wanted to confirm before I make any more changes. 

Also, do we need to update this line -- since it rolls back pretty far?

```
$ pip install --upgrade streamlit==0.37 # <- To downgrade Streamlit!
```